src/ServiceLinker/readme.md

```diff
@@ -0,0 +1,138 @@
+<!-- region Generated -->
+# Az.ServiceLinker
+This directory contains the PowerShell module for the ServiceLinker service.
+
+---
+## Status
+[![Az.ServiceLinker](https://img.shields.io/powershellgallery/v/Az.ServiceLinker.svg?style=flat-square&label=Az.ServiceLinker "Az.ServiceLinker")](https://www.powershellgallery.com/packages/Az.ServiceLinker/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.2.3 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.ServiceLinker`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+# readme.azure.noprofile.md is the common configuration file
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+# You need to specify your swagger files here.
```

Comment:
Please add commit id here to lock the version of servicelinker.json. Please refer to: https://github.com/Azure/azure-powershell/blob/12c25f919424b736555b5fd6c2a59e01bef64b9c/src/ContainerInstance/README.md?plain=1#L33

Reply:
added